[STAL-2778] Add note about using the `code_analysis_read` scope (#46)
DataDog/datadog-static-analyzer-github-action: README.md

          enable_performance_statistics: false
```

> You **must** set your Datadog API and application keys as [secrets in your GitHub repository][4] whether at the organization or repository level. Please add the `code_analysis_read` scope to your Datadog application key requ the. For more information, see [API and Application Keys][2].

Reviewer: remove `requ the` ?

> You **must** set your Datadog API and application keys as [secrets in your GitHub repository][4] whether at the organization or repository level. Please add the `code_analysis_read` scope to your Datadog application key requ the. For more information, see [API and Application Keys][2].

Reviewer 2: ```suggestion
You **must** set your Datadog API and application keys as [secrets in your GitHub repository][4] whether at the organization or repository level. You must also ensure that you add the `code_analysis_read` scope to your Datadog application key. For more information, see [API and Application Keys][2].
```

> You **must** set your Datadog API and application keys as [secrets in your GitHub repository][4] whether at the organization or repository level. You must also ensure that you add the `code_analysis_read` scope to your Datadog application key. For more information, see [API and Application Keys][2].

Reviewer 3: ```suggestion
You **must** set your Datadog API and application keys as [secrets in your GitHub repository][4] whether at the organization or repository level. Ensure that you add the `code_analysis_read` scope to your Datadog application key. For more information, see [API and Application Keys][2].
```
